ui/react-ui/src/components/routes/home/Home.js

```diff
@@ -95,7 +102,7 @@ class Home extends React.Component {
                       return (
                         <input 
                           className='param-input' 
-                          id={endpoint.key} type='text' 
+                          id={`${endpoint.key}-${param.key}`} type='text' 
```

Comment:
thanks for updating this line. also as I mentioned in the slack channel, if you replace the line 62 with this 
`if (param.value) updatedEndpoint = updatedEndpoint.replace(param.regex, param.value);`
the bug that we have in there to add the params in the endpoint will be fixed

Reply:
Should be fixed.